docs/src/main/paradox/index.md

```diff
@@ -5,30 +5,30 @@ This table lists all Akka modules that are part of the Akka $akka-dependencies.m
-| Akka Persistence plugin for Cassandra | $akka-persistence-cassandra.version$ | [Documentation](https://doc.akka.io/docs/akka-persistence-cassandra/current/)                                                                                                                                                                                                                                          |
-| Akka Persistence plugin for JDBC      | $akka-persistence-jdbc.version$      | [Documentation](https://doc.akka.io/docs/akka-persistence-jdbc/current/)                                                                                                                                                                                                                                               |
-| Akka Persistence plugin for R2DBC     | $akka-persistence-r2dbc.version$     | [Documentation](https://doc.akka.io/docs/akka-persistence-r2dbc/current/)                                                                                                                                                                                                                                              |
-| Akka Projections                      | $akka-projections.version$           | [Documentation](https://doc.akka.io/docs/akka-projection/current/)                                                                                                                                                                                                                                                     |
-| Akka Management                       | $akka-management.version$            | [Documentation](https://doc.akka.io/docs/akka-management/current/)                                                                                                                                                                                                                                                     |
-| Akka Diagnostics                      | $akka-diagnostics.version$           | [Documentation](https://doc.akka.io/docs/akka-diagnostics/current/)                                                                                                                                                                                                                                                    |
+| Akka (core)                           | $akka.version$                       | [Documentation](https://doc.akka.io/libraries/akka-core/current/) [JavaDoc](https://doc.akka.io/japi/akka/current/index.html) [ScalaDoc](https://doc.akka.io/api/akka/current/akka/)                                                                                                                                             |
```

Comment:
those links are not active yet, so if we merge this we must create a branch and release akka-dependencies from that until we have `libraries/` in place